vm/src/class.rs

```diff
@@ -60,6 +60,7 @@ pub trait PyClassDef {
     const TP_NAME: &'static str;
     const DOC: Option<&'static str> = None;
     const BASICSIZE: usize;
+    const UNHASHABLE: bool;
```

Comment:
```suggestion
    const UNHASHABLE: bool = false;
```
default value can be placed here